### ISSUE TITLE
sprit bear/other bear auto quests

### ISSUE THREAD

**Issue by jowsaw2** [NatroTeam/NatroMacro#40]
### Have you searched the existing suggestions?

No

### What is your suggestion?

what: auto does quests for the bears including th ereapeating ones

why: cause many mid game players struggle with spirit bear quests and auto quest couldl be very helpful

### How will this improve Natro Macro?

why: because alot of people don't want to waste times doing quests

### What is your Discord username?

jowsaw

**Comment by AWKYTB** [comment]
tysm

**Comment by jowsaw2** [comment]
as you said people already use macros to do main quest so i don't see your point

**Comment by zspz** [comment]
Yes, I would like to confirm that non-repeatable quests will **never** be added to the macro. Natro Macro is intended to be a tool that aids your progress when you're not available (e.g. sleeping). Adding non-repeatable quests will not be of best interest to anyone.

You can use Natro Macro to **help** finishing the objectives for non-repeatable quests, but **doing it for you** would take the fun out of the game.
